web/packages/teleport/src/Discover/SelectResource/Tile.tsx

```diff
@@ -30,17 +33,28 @@ import {
 
 import { getResourcePretitle } from '.';
 import { DiscoverIcon } from './icons';
-import { SelectResourceSpec } from './resources';
+import { SelectResourceSpec } from './resources/types';
 
 export function Tile({
   resourceSpec,
+  isPinTile = false,
   onChangeShowApp,
   onSelectResource,
+  onChangePinGuide,
+  pinningSupport,
 }: {
+  /**
+   * if true, renders slightly different tile design
+   */
+  isPinTile?: boolean;
```

Comment:
this is a bit cryptic. _what_ is different about it?

Reply:
clarified (it just renders a larger tile)